
Support "--enable-modules=none" to build an httpd binary with 
no optional modules enabled.

Upstream-Status: committed to trunk, r357168

--- httpd-2.1.10/acinclude.m4.disablemods
+++ httpd-2.1.10/acinclude.m4
@@ -289,14 +289,19 @@
 
   AC_ARG_ENABLE(modules,
   APACHE_HELP_STRING(--enable-modules=MODULE-LIST,Space-separated list of modules to enable | "all" | "most"),[
-    for i in $enableval; do
-      if test "$i" = "all" -o "$i" = "most"; then
-        module_selection=$i
-      else
-        i=`echo $i | sed 's/-/_/g'`
-        eval "enable_$i=yes"
-      fi
-    done
+    if test "$enableval" = "none"; then
+       module_default=no
+       module_selection=none
+    else
+      for i in $enableval; do
+        if test "$i" = "all" -o "$i" = "most"; then
+          module_selection=$i
+        else
+          i=`echo $i | sed 's/-/_/g'`
+          eval "enable_$i=yes"
+        fi
+      done
+    fi
   ])
   
   AC_ARG_ENABLE(mods-shared,
